src/Tizen.Multimedia.Util/ThumbnailExtractor/ThumbnailExtractor.cs

```diff
@@ -246,6 +246,8 @@ private static Native.ThumbnailExtractCallback GetCallback(TaskCompletionSource<
         /// </summary>
         /// <remarks>
         /// The size of generated thumbnail will be 320x240.<br/>
+        /// But, if input image have different ratio with 320x240 (approximately 1.33:1),<br/>
+        /// thumbnail could be generated with different size based on short axis of default size.<br/>
```

Comment:
if you specify whether a image is cropped or stretched, it would be better.

Reply:
I'll update description more precisely.